getting_started/first_2d_game/03.coding_the_player.rst

```diff
@@ -91,9 +91,9 @@ value written in the script.
 
 .. warning:: If you're using C#, you need to (re)build the project assemblies
              whenever you want to see new export variables or signals. This
-             build can be manually triggered by clicking the word "Mono" at the
-             bottom of the editor window to reveal the Mono Panel, then clicking
-             the "Build Project" button.
+             build can be manually triggered by clicking the word "MSBuild" at the
+             bottom of the editor window to reveal the MSBuild Panel, then clicking
+             the "Build" button.
```

Comment:
Maybe mention the build button in the top right as well? 
![image](https://user-images.githubusercontent.com/10944644/221137061-ab08f31d-5a82-48e9-ab1b-a991dcf158a5.png)